proto/bragi/common.proto

```diff
@@ -75,7 +75,13 @@ message Tournament {
 message Competitor {
   string id = 1; // id is urn in format e.g. 'od:team:123' or 'od:player:123'
   string name = 2;
-  optional bool home_team = 3;
+  TeamType team_type = 3;
+}
+
+enum TeamType {
```

Comment:
Looks promising. Could be extended to
```
message Competitor {
    enum Type {
        ...
        COMPETITOR_TYPE_HOME_TEAM = 1;
        COMPETITOR_TYPE_AWAY_TEAM = 2;
        COMPETITOR_TYPE_TEAM = 3;    // in future e.g. for racing markets without home/away
        COMPETITOR_TYPE_PLAYER = 4;  // in future
    }
}
```

@pmachan @kafkapre WDYT?

Reply:
`enum TeamType` is confusing, its used inside of Competitor and Competitor can be player or something else